cmd/image-builder/main.go

```diff
@@ -248,6 +229,11 @@ func prepareADOTemplateParameters(options options) (adopipelines.OCIImageBuilder
 		templateParameters.SetImageTags(options.tags.String())
 	}
 
+	isGhAction := os.Getenv("GITHUB_ACTIONS")
```

Comment:
This information is already read from env var and added to the options struct. You added functions in config.go file for this. Can we use the value from options instead reading again from env? This will improve consistency in the code, as we will always read from options once the value is get from env.

Reply:
Yes, I probably miss that line when changing the way we handle it.